build/windows/install.nsi

```diff
@@ -3,7 +3,7 @@
 !define APP_NAME "Backrest"
 !define COMP_NAME "garethgeorge"
 !define WEB_SITE "https://github.com/garethgeorge/backrest"
-!define VERSION "00.00.00.00"
+!define VERSION "1.6.1.0"
```

Comment:
Note: this will actually be out of date immediately with the next version, the first version of this installer will ship as 1.6.1. 

I'll merge as is since it's not really a blocker, but I think I need to followup up re: automatically passing in the version from the release workflow.

Reply:
It turns out NSIS is quite flexible and already had such a scenario covered. I will submit a new PR to have the installer grab the version from the changelog.

Reply:
Neat! Is that a built in capability of NSIS? My original thinking had been to stamp the version based on the git tag’s on the current commit — the release workflow tags a release commit with the current version.

Reply:
Yes, built-in functionality. Given the format and structure of changelog.md, this should work well.